secure_message/authorization/authorizer.py

```diff
@@ -24,7 +24,7 @@ def can_user_view_message(user, message):
         if user.user_uuid == message['msg_from']:
             return True
 
-        if user.user_uuid in message['msg_to'] and 'DRAFT_INBOX' not in message['labels']:
+        if user.user_uuid in message['msg_to']:
```

Comment:
the whole of this could be simplified for to:

return user.is_internal or user.user_uuid in [message['msg_from'] + message['msg_to']]

Only styinge though, so feel free to reject